src/vivarium/framework/resource.py

```diff
@@ -106,20 +110,20 @@ def __str__(self) -> str:
 class ResourceManager(Manager):
     """Manages all the resources needed for population initialization."""
 
-    def __init__(self):
+    def __init__(self) -> None:
         # This will be a dict with string keys representing the the resource
         # and the resource group they belong to. This is a one to many mapping
         # as some resource groups contain many resources.
-        self._resource_group_map = {}
+        self._resource_group_map: dict[str, ResourceGroup] = {}
```

Comment:
Can you convert these inline comments to docstrings?

Reply:
I thought about doing that, but they won't show up b/c they're private anyway. Are you suggesting just so it's easier to know which comment goes w/ wihich attr?